docs/metatype.dev/blog/2024-09-26-introducing-grpc-runtime/index.mdx

```diff
@@ -0,0 +1,92 @@
+import BlogIntro from "@site/src/components/BlogIntro";
+
+# Introducing gRPC Runtime
+
+<BlogIntro text="We're excited to announce the new gRPC Runtime feature in Metatype, further enhancing our platform's ability to create versatile and powerful backends through typegraphs." />
+
+## What is Metatype?
+
+Before we dive into the gRPC Runtime, let's recap what Metatype is all about:
+
+Metatype is a platform that allows developers to create typegraphs - strongly-typed, composable backend structures. With Metatype, you can:
+
+- Construct solid, type-safe backends
+- Leverage multiple runtimes for different services and protocols
+- Expose your typegraphs via GraphQL or HTTP endpoints
+
+## Introducing gRPC Runtime
+
+The new gRPC Runtime is the latest addition to Metatype's suite of runtimes, joining existing options like the HTTP runtime. This expansion allows you to incorporate gRPC services into your typegraphs, further enhancing the versatility of your Metatype-powered backends.
+
```

Comment:
it is missing "what is the grpc" with a diagram imo here, so people not familar may understand why grpc is important, then see how to use it with metatype